docs/WIRELESS.md

```diff
@@ -0,0 +1,162 @@
+# Support for wireless connectivity in EVE
+
+Given that the biggest use case for EVE is in Edge Computing and IoT it is no surprise that wireless connectivity plays a major role in connecting edge applications. Currently EVE supports 2 kinds of wireless networking devices: WiFi cards and Cellular GSM Modems. When it comes to utilizing either of these EVE relies on its controller to supply two critical pieces of information:
+
+* Name of the wireless network to be joined ([SSID](https://en.wikipedia.org/wiki/Service_set_(802.11_network)#SSID) for WiFi and [APN](https://en.wikipedia.org/wiki/Access_Point_Name) for Cellular).
+* Pre-shared secret required to joining the network ([WPA/WPA2/WEP](https://en.wikipedia.org/wiki/Wi-Fi_Protected_Access) and [username/password](https://www.digi.com/resources/documentation/Digidocs/90001399-13/tasks/t-set-up-and-configure-mobile-connectivity.htm) for Cellular).
+
+This dependency on a controller creates a bit of a chicken-and-an-egg problem when the wireless connectivity itself happens to be the only means of reaching the controller. In those cases EVE uses pre-loaded controller configuration to bootstrap the entire process.
+
+The rest of this document will be focused on details of wireless support in EVE. It must be noted that currently we're relying on a patchwork of different open source projects to do that, but we're also looking at things like [oFono](https://en.wikipedia.org/wiki/OFono) and [Android's RIL](https://en.wikipedia.org/wiki/Radio_Interface_Layer) for a more hollistic support.
+
+## WiFi
+
+In general, WiFi support in EVE is pretty straightforward and largely depends on enabling required driver in the Linux kernel and finding an appropriate firmware binary blob to be loaded by the device driver. Please refer to our [new hardware bringup](HARDWARE-BRINGUP.md) document for more details on the former and make sure to checkout our [firmware package](../pkg/firmware) for the later.
+
+## Cellular GSM Modems
+
+Compared to WiFi, cellular GSM modems present a major challenge. This is partially understandable, since radio regulations in different countries tend to be partially incompatible and at times [simply insane](https://www.afcea.org/content/disruptive-design-have-you-seen-flurry-acts-5g-security). All of this conspires to create a technology landscapte where modem manufacturers are forced to produce highly capable and highly configurable units that take onto very different personalities when loaded with firmware and given their final configuration.
```

Comment:
"simple insane" - I love this!

Reply:
Typo: landscapte
Might make sense to spell check the document

Reply:
I did ;-) but it was an ispell(1) so.... ;-)